src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
do not create new Random() each time
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/3211b71d-d143-4d1a-8e2a-249a5d3a4739)